src/category_theory/preadditive/injective.lean

```diff
@@ -174,6 +174,28 @@ begin
   exact functor.preserves_epimorphisms.iso_iff (coyoneda.obj_op_op _)
 end
 
+section adjunction
+
+ open category_theory.functor
+
+ universes v₁ v₂ u₁ u₂
+
+ variables {𝓐 : Type u₁} {𝓑 : Type u₂} [category.{v₁ u₁} 𝓐] [category.{v₂ u₂} 𝓑]
```

Comment:
There's no need to introduce completely new universes and categories here. Just change the universe declaration in line 24 to `universes v₁ v₂ u₁ u₂` and adapt the declaration of `C` accordingly. Down here it then suffices to declare `{D : Type u₂} [category.{v₂} D]`. Please note that in mathlib we generally call categories `C`, `D`, ... without any fancy unicode font.

Reply:
done